Lesson18/sahra/script.js

```diff
@@ -0,0 +1,169 @@
+// 1. Write a function that takes an array of integers and returns the sum of all elements.
+// Input: [1, 2, 3, 4]
+// Output: 10
+const integersEx1 = [1, 2, 3, 4];
+const integersEx1V2 = [10, 2, 30, 4];
+function sum(myArr){
+  let sum = 0;
+  for(let i = 0; i < myArr.length; i++){
+    sum += myArr[i];
+  }
+  return sum;
+}
+const result = sum(integersEx1V2);
+console.log(result);
+
+console.log('----------------------------');
+// 2. Write a function that takes an array of integers and returns the average of all elements.
+//Input: [1, 2, 3, 4]
+// Output: 2.5
+function numbers(myArr){
+  let sum = 0;
+  for(let i=0; i < myArr.length; i++){
+    sum += myArr[i];
+    sum /= myArr.length;
```

Comment:
You need to do the division after the loop (after }) otherwise it won't work correctly